repo/nirepo.yml

```diff
@@ -0,0 +1,1783 @@
+      x-ni-privilege: Write
+      x-ni-operation: deleteFeeds
+      tags: [feeds]
+      summary: Remove a feed
+      description: Asynchronously removes a feed. Returns the ID of a resource which can be queried to determine the status of the operation.
+      parameters:
+        - in: query
+          name: deleteUnreferencedPackages
+          type: boolean
+          description: If true, any packages in the feed that are no longer referenced will be deleted.
+          default: true
+      responses:
+        202:
+          $ref: '#/responses/JobIdResponse'
+        401:
+          $ref: '#/definitions/Unauthorized'            
+        default:
+          $ref: '#/definitions/Error'
+  /v1/feeds/{id}/clone:
+    parameters:
+      - in: path
+        name: id
+        description: The id of this resource.
+        type: string
+        required: true
+        x-example: "5c5d49f2781e2b2cd045267b"
+    post:
+      x-ni-privilege: Write
+      x-ni-operation: cloneFeed
+      tags: [feeds]
+      summary: Clone a feed
+      description: Asynchronously clones a feed. Returns the ID of a resource which can be queried to determine the status of the operation.
+      parameters:
+        - in: body
+          name: feedRequestData
+          schema:
+            type: object
+            properties:
+              name:
+                type: string
+                description: A string representing the name for the cloned feed.
+                example: Feed Name
+              description:
+                type: string
+                description: A description for the cloned feed.
+                example: Feed Description
+      responses:
+        202:
+          $ref: '#/responses/JobIdResponse'
+        400:
+          $ref: '#/definitions/Error'
+        401:
+          $ref: '#/definitions/Unauthorized'            
+        default:
+          $ref: '#/definitions/Error'
+  /v1/feeds/{id}/packages:
+    parameters:
+      - in: path
+        name: id
+        description: The id of this resource.
+        type: string
+        required: true
+        x-example: "5c5d49f2781e2b2cd045267b"
+    get:
+      x-ni-privilege: Read
+      x-ni-operation: getPackages
+      tags: [feeds, packages]
+      summary: Get all packages in a feed
+      description: Gets all of the packages in the service's package pool.
+      parameters:
+        - in: query
+          name: omitAttributes
+          type: boolean
+          description: If true, the "Attributes" dictionary for the packages will be omitted.
+          default: false
+        - in: query
+          name: omitFeedReferences
+          type: boolean
+          description: If true, the array of feed references will be omitted from each package.
+          default: false
+        - in: query
+          name: returnUnreferencedOnly
+          type: boolean
+          description: A query parameter that determines if only packages without referencing feeds should be returned.
```

Comment:
Can we replace "if" with "whether". Please implement this strategy with the remaining descriptions for booleans.